context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/style.ts

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
```

Comment:
```suggestion
import { styled } from '@mui/material/styles';
```

Importing the styled utility from MUI will make it more straightforward for us to eventually upgrade to a more performant styling approach (likely https://github.com/mui/pigment-css once it's mainlined in MUI 6). It also makes it easier to access theme values in the styles via TypeScript hints.